src/transceivercontroller/TransceiverController.ts

```diff
@@ -62,6 +62,17 @@ export default interface TransceiverController {
     videosToReceive: VideoStreamIdSet
   ): number[];
 
+  /**
+   * Get the associated MID for a given stream ID, either set via `updateVideoTranceivers`
```

Comment:
Is `updateVideoTranceivers` an existing API?

Reply:
yes.  These APIs are compatible with the previous subscribe path, which is still hit for anything that isn't a simulcast stream switch.